test/assemblers/assemble_contributions_summary_test.rb

```diff
@@ -45,10 +61,8 @@ class AssembleContributionsSummaryTest < ActiveSupport::TestCase
           title: "All Tracks",
           icon_url: "ICON",
           categories: [
-            # TODO: Reputation
-            { id: :publishing, reputation: 0, metric_full: "6 solutions published", metric_short: "6 solutions" },
-            # TODO: Reputation
-            { id: :mentoring, reputation: 0, metric_full: "9 students mentored", metric_short: "9 students" },
+            { id: :publishing, reputation: 10, metric_full: "5 solutions published", metric_short: "5 solutions" },
+            { id: :mentoring, reputation: 45, metric_full: "9 students mentored", metric_short: "9 students" },
```

Comment:
Apparently I just never found a way to even implement this before in a time that was sane. (Apparently 22s was sane so goodness knows how long this took...)